cplusplus.json

```diff
@@ -953,5 +953,52 @@
+  "imports": ["fplus/"], "examples": ["https://github.com/Dobiasd/FunctionalPlus/tree/master/examples"]},
+  {"id": "cpp.glib", "name": "GLib", "tech": ["app-arch"], "tags": ["core-utils"], "repo": "https://gitlab.gnome.org/GNOME/glib/tree/master",
+  "imports": ["glib.h", "glib-"], "examples": ["https://gitlab.gnome.org/GNOME/glib/tree/master/tests"]},
+  {"id": "cpp.juce", "name": "JUCE", "tech": ["audio"], "tags": ["cross-platform"], "repo": "WeAreROLI/JUCE",
+  "imports": ["juce_"], "examples": ["https://github.com/WeAreROLI/JUCE/tree/master/examples"]},
+  {"id": "cpp.libphenom", "name": "libPhenom", "tech": ["app-arch"], "tags": ["eventing"], "repo": "facebook/libphenom",
+  "imports": ["phenom/"], "examples": ["https://github.com/facebook/libphenom/tree/master/examples"]},
+  {"id": "cpp.libsourcey", "name": "LibSourcey", "tech": ["networking"], "tags": ["evented"], "repo": "sourcey/libsourcey",
+  "imports": ["scy/"], "examples": ["https://sourcey.com/libsourcey/examples/"]},
+  {"id": "cpp.libu", "name": "LibU", "tech": ["utils"], "tags": ["misc"], "repo": "koanlogic/libu",
+  "imports": ["u/libu.h"], "examples": ["https://github.com/koanlogic/libu/tree/master/example"]},
+  {"id": "cpp.loki", "name": "Loki", "tech": ["app-arch"], "tags": ["design-patterns"], "repo": "https://sourceforge.net/p/loki-lib/code/HEAD/tree/trunk/",
+  "imports": ["loki/"], "examples": ["https://sourceforge.net/p/loki-lib/code/HEAD/tree/trunk/test/"]},
+  {"id": "cpp.mili", "name": "MiLi", "tech": ["app-arch"], "tags": ["collection", "misc"], "repo": "https://bitbucket.org/fudepan/mili/src",
+  "imports": ["mili/"], "examples": ["https://bitbucket.org/fudepan/mili/src/6ec9fce00b988a7759a80e97ada52533f89fd2fb/examples/?at=default"]},
+  {"id": "cpp.open-frameworks", "name": "openFrameworks", "tech": ["graphics-gaming"], "tags": ["creative-coding"], "repo": "openframeworks/openFrameworks",
+  "imports": ["ofMain.h", "of"], "examples": ["https://github.com/openframeworks/openFrameworks/tree/master/examples"]},
+  {"id": "cpp.root", "name": "ROOT", "tech": ["big-data"], "tags": ["analyzing", "visualization"], "repo": "root-project/root",
+  "imports": ["root/"], "examples": ["https://github.com/root-project/root/tree/master/tutorials"]},
+  {"id": "cpp.stxxl", "name": "STXXL", "tech": ["big-data"], "tags": ["stl-kind"], "repo": "stxxl/stxxl",
+  "imports": ["stxxl/"], "examples": ["https://github.com/stxxl/stxxl/tree/master/examples"]},
+  {"id": "cpp.tbox", "name": "tbox", "tech": ["app-arch"], "tags": ["cross-platform-clib"], "repo": "tboox/tbox",
+  "imports": ["tbox/tbox.h"], "examples": ["https://github.com/tboox/tbox/tree/master/src/demo"]},
+  {"id": "cpp.ultimatepp", "name": "Ultimate++", "tech": ["gui"], "tags": ["cross-platform"], "repo": "ultimatepp/mirror",
+  "imports": [], "examples": ["https://github.com/ultimatepp/mirror/tree/master/examples"]},
```

Comment:
no imports? is it working via #pragmas or something?

Reply:
No, but too many directives to be included. Let me add them